### PR TITLE
DOC-8057 added scopes for array function and bitwise page

### PR DESCRIPTION
**Pull request by sixthcodebrewer**

### CODE REVIEW

File: modules/n1ql/pages/n1ql-language-reference/arrayfun.adoc [hunk 1]
```diff
@@ -170,15 +169,15 @@ If one of the arguments is `MISSING`, it returns `MISSING`.
 If the input [.var]`expr` is not an array, it returns `NULL`.
 
 === Example
+// this is giving wrong output
```
Comment (simon-dew) [inline modules/n1ql/pages/n1ql-language-reference/arrayfun.adoc]: Checked and yes, the results given in the doc are wrong. Please update the result on line 189-190 to `"sorted-position": 0` and delete this comment.

File: modules/n1ql/pages/n1ql-language-reference/arrayfun.adoc [hunk 2]
```diff
@@ -948,7 +938,7 @@ Prepend "Dave Smith" to the front of the `public_likes` array.
 ----
 SELECT ARRAY_PREPEND("Dave Smith",t.public_likes) AS prepend_val
 FROM `travel-sample` t
```
Comment (simon-dew) [inline modules/n1ql/pages/n1ql-language-reference/arrayfun.adoc]:
```suggestion
FROM `travel-sample`.inventory.hotel t
```

File: modules/n1ql/pages/n1ql-language-reference/arrayfun.adoc [hunk 3]
```diff
@@ -948,7 +938,7 @@ Prepend "Dave Smith" to the front of the `public_likes` array.
...
-WHERE type="hotel"
+.inventory.hotel
```
Comment (simon-dew) [inline modules/n1ql/pages/n1ql-language-reference/arrayfun.adoc]:
```suggestion
```

File: modules/n1ql/pages/n1ql-language-reference/arrayfun.adoc [hunk 4]
```diff
@@ -948,7 +938,7 @@ Prepend "Dave Smith" to the front of the `public_likes` array.
...
 LIMIT 1;
```
Comment (simon-dew) [inline modules/n1ql/pages/n1ql-language-reference/arrayfun.adoc:941]: So the whole thing should be
```
SELECT ARRAY_PREPEND("Dave Smith",t.public_likes) AS prepend_val
FROM `travel-sample`.inventory.hotel t
LIMIT 1;
```

**Review by simon-dew — CHANGES_REQUESTED** [review]
Just a couple of comments to address.

(Incidentally @amarantha-k the document also has examples using the `default` keyspace and the `contacts` keyspace, neither of which exist and which therefore cannot be reproduced. It's outside the scope of this change, but just to note we will probably need a separate effort to deal with examples like this.)

File: modules/n1ql/pages/n1ql-language-reference/arrayfun.adoc (hunk 1 above)
Comment (sixthcodebrewer) [reply modules/n1ql/pages/n1ql-language-reference/arrayfun.adoc]: thank you changed it

**Review by simon-dew — APPROVED** [review]

**Comment by simon-dew** [comment]
Made one last minor change myself. 👍 LGTM

**Pull request merged**